### PR TITLE
add bhnt 72 hacks fixes #254

### LINKED ISSUE #254: bhnt 72 add hacks
1. AFL
1. webRTC beaker 
1. dotfile rage by chr6stpr
1. Quadrobot
1. DebLib by Henry addo 
1. Crypto yield curve by Max
1. Hexliterate by ssimono
1. Infinito by KevinFranz

### PR DESCRIPTION
**Pull request by lucasrangit**

### CODE REVIEW

File: _posts/2019-06-25-off-by-one.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+* AFL
+* [webRTC beaker](https://github.com/D1plo1d/tegh-cli) by [Rob Gilson](https://github.com/D1plo1d)
+* [dotfile rage](https://gist.github.com/chr5tphr/506e4360f7569dcb1495cf63aaff9c90) by [Christopher](https://github.com/chr5tphr)
+* [Quadruped ROS2 Bot](https://hackaday.io/project/165423-quadruped-ros2-bot) by [Mario](https://github.com/qwqw330/)
+* [DebLibs](https://github.com/hellofresh/deblibs-gradle-plugin) by [Henry Addo](https://github.com/eyedol)
+* Crypto yield curve by Max
+* [Hexliterate](https://github.com/ssimono/hexliterate) by [Simon Alfassa](https://github.com/ssimono)  - **Hack of the month**
+* [https://github.com/KevinFrantz/infinito](infinito) by [https://github.com/KevinFrantz](Kevin Frantz)
+
```
Comment (ligi) [inline _posts/2019-06-25-off-by-one.md]: can you remove this line?

File: _posts/2019-06-25-off-by-one.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Berlin Hack & Tell \#72 - Off By One++
+date: 2019-06-25
+time: '18:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/Berlin-Hack-and-Tell/events/262505358/
+---
+
+* AFL
```
Comment (ligi) [inline _posts/2019-06-25-off-by-one.md]: is there a bit more info on this?

Comment (lucasrangit) [reply _posts/2019-06-25-off-by-one.md]: I was late and missed that one. @MarlonSchultz , do you have any more details?

File: _posts/2019-06-25-off-by-one.md (hunk 1 above)
Comment (lucasrangit) [reply _posts/2019-06-25-off-by-one.md]: Do you mean fix? I fixed it just now.

Comment (ligi) [reply _posts/2019-06-25-off-by-one.md]: no - I mean remove the blank line on the end

Comment (lucasrangit) [reply _posts/2019-06-25-off-by-one.md]: Oh I see. Removed.

**Pull request merged**